pre_commit_hooks/check_number_of_lines_count.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+import argparse
+from typing import Optional
+from typing import Sequence
+
+
+def main(argv: Optional[Sequence[str]] = None) -> int:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('filenames', nargs='*')
+    parser.add_argument(
+        '--max-lines',
+        default=30,
+        type=int,
+        action='store',
+        help='Maximum allowable number of lines (default: 30)',
```

Comment:
The default is already shown by argparse

```suggestion
        help='Maximum allowed number of lines',
```